bin/dx-unpack

```diff
@@ -38,7 +38,7 @@ while [[ $# > 0 ]]; do
         output="$2"
         shift 2
     else
-        A="$A $1"
+        A="$A \"$1\""
```

Comment:
You could also consider inlining the function "unpack" or converting this whole thing to a Python script (don't know if either of those would make things any simpler... but it's something to evaluate to avoid bash nonsense).